.gitignore

```diff
@@ -21,6 +21,7 @@
 .project
 .settings/
 .vscode/
+.dart/
```

Comment:
@DanTup Does VS Code actually produce this directory? Should it be ignored?

Reply:
There's nothing I'm aware of in VS Code or the Dart/Flutter extensions for VS code that creates `.dart` directory.